src/lib/utils/loadstor.h

```diff
@@ -82,6 +83,56 @@ inline constexpr uint64_t make_uint64(
            (static_cast<uint64_t>(i6) << 8) | (static_cast<uint64_t>(i7)));
 }
 
+/**
+* Load a big-endian unsigned integer
```

Comment:
This sounds like the unsigned integer itself must be stored big-endian (or little-endian). Maybe we want to chose a wording like: "Load an unsigned integer into a byte range in big-endian order".

Reply:
Yeah, sorry for the confusion. I meant that it sounds like the integer itself, which is the output of this function, is big-endian. However, of course, the input bytes are interpreted in big-endian order. But you are right. Probably it's clear enough 👍